java/client/src/org/openqa/selenium/devtools/network/events/LoadingFailed.java

```diff
@@ -0,0 +1,130 @@
+package org.openqa.selenium.devtools.network.events;
+
+import org.openqa.selenium.devtools.network.types.BlockedReason;
+import org.openqa.selenium.devtools.network.types.RequestId;
+import org.openqa.selenium.devtools.network.types.ResourceType;
+import org.openqa.selenium.json.JsonInput;
+
+/**
+ * Object for storing Network.loadingFailed response
+ */
+public class LoadingFailed {
+
+  /**
+   * Request identifier
+   */
+  private final RequestId requestId;
+
+  /**
+   * MonotonicTime
+   */
+  private final Number timestamp;
+
+  /**
+   * Resource type
+   */
+  private final ResourceType type;
+
+  /**
+   * User friendly error message
+   */
+  private final String errorText;
+
+  /**
+   * True if loading was canceled
+   */
+  private final Boolean canceled;
+
+  /**
+   * The reason why loading was blocked, if any
+   */
+  private final BlockedReason blockedReason;
+
+  public LoadingFailed(RequestId requestId, Number timestamp,
+                       ResourceType resourceType, String errorText, Boolean canceled,
```

Comment:
Hi @shs96c ,
The Boolean is in case of null usage.
When the value is optional then the values might be true\false or null.
this is why we have added such Object